lib/flux/app-state.js

```diff
@@ -696,6 +696,79 @@ export const actionMap = new ActionMap({
         tags: { $set: tags },
       });
     },
+
+    loadPreferences: {
+      creator({ callback, preferencesBucket }) {
+        return dispatch => {
+          const objectKey = 'preferences-key';
+
+          preferencesBucket.get(objectKey, (e, preferences) => {
+            const preferencesData = preferences ? preferences.data : {};
```

Comment:
Suggestion: This might be a nice place to use `lodash`'s `get` method: https://lodash.com/docs/4.17.10#get

something like `get(preferences, 'data.analytics_enabled', true)`?

Reply:
Thanks! I'm not yet familiar with all the lodash functions, so please keep these suggestions coming 🙂

Reply:
Nice! Change looks good 👍